r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -3772,22 +3772,24 @@ test_that("hms::as_hms", {
     int = c(30L, 75L),
```

Comment:
I can't comment on the line above, but should we change that `hms_string` to something like: `hms_string = c("0:7:45.42", "12:34:56.57")` so that we get coverage for that?

Or maybe we want to add in tens or ones digits to the `datetime` column?

Reply:
It's complicated with character strings as [we don't support subseconds in `strptime()`](https://github.com/apache/arrow/issues/20146).  I've added another test to be more explicit about this.